cypress/e2e/CMS/$CancerTypeHomePage/index.js

```diff
@@ -163,5 +140,121 @@ And('user selects {string} from draft {string} dropdown', (dropdown, option) =>
+    cy.get("div[class*='cthp-research'] div[class*='label']").then($el => {
+        summaryPageTitle = $el[0].innerText
+    })
+})
+
+Then('the Cancer Research List page title is matching the earlier selected Cancer Research List page title', () => {
+    cy.get("h1").should('include.text', summaryPageTitle)
+
+})
+
+And('user clicks on {string} dropdown', (dropdown) => {
+    cy.get(`div[class='more-info list cthp-dropdown'] label:contains("${dropdown}")`).click({ force: true })
+})
+
+And('the following more info links are displayed', (dataTable) => {
+    for (const { title, url } of dataTable.hashes()) {
+        const replacedTestSiteSection = url.replace("{TEST_SITE_SECTION}", siteSection);
+        cy.get(`ul a[class='title']:contains("${title}")`).should('be.visible').and('have.attr', 'href', replacedTestSiteSection)
+    }
+})
+
+And('cthp causes card has a link {string} with href {string}', (linkText, href) => {
+    const replacedTestSiteSection = href.replace("{TEST_SITE_SECTION}", siteSection);
+    cy.get("div[class='equalheight bgWhite cthp-causes'] a").should('have.text', linkText)
+    cy.get("div[class='equalheight bgWhite cthp-causes'] a").should('have.attr', 'href', replacedTestSiteSection)
+})
+
+And('cthp survival card has a link {string} with href {string}', (title, href) => {
+    cy.get(`div[class='equalheight bgWhite cthp-survival'] a:contains("${title}")`).should('be.visible').and('have.attr', 'href', href)
+})
+
+And('cthp screening card has a link {string} with href {string}', (linkText, href) => {
+    const replacedTestSiteSection = href.replace("{TEST_SITE_SECTION}", siteSection);
+    cy.get("div[class='equalheight bgWhite cthp-screening'] a").should('have.text', linkText)
+    cy.get("div[class='equalheight bgWhite cthp-screening'] a").should('have.attr', 'href', replacedTestSiteSection)
+})
+
+And('cthp general card has description that is not empty', () => {
+    cy.get("div[class='equalheight bgWhite cthp-general'] p").should('be.visible')
+})
+And('cthp general card has multiple links', () => {
+    cy.get("div[class='equalheight bgWhite cthp-general'] p").should('be.visible')
+})
+
+And('cthp genetics card reads {string}', (title) => {
+    cy.get(`div[class='equalheight bgWhite cthp-genetics'] div:contains("${title}")`).should('be.visible')
+})
+
+And('cthp research card has multiple links', () => {
+    cy.get(`div[class='equalheight bgWhite cthp-research'] a`).should('be.visible')
+})
+
+And('user clicks {string} link inside cthp research card', (option) => {
+    cy.get(`a:contains("${option}")`).click({ force: true })
+})
+
+And('user removes Video from {string} card area', (section) => {
+    cy.get(`div:contains("${section}")`).parent().find("input[name*='field_cthp_video_remove']").click({ force: true })
+})
+
+And('user selects {int} Video from the list of videos', (num) => {
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find('input.form-checkbox').first().eq(num - 1).check()
```

Comment:
this is redundunt 
.first().eq(num - 1) - first() grabs the first element, and then you pass an index - I think you need to remove first here